cmake/kconfig.cmake

```diff
@@ -199,6 +199,7 @@ endif()
 
 execute_process(
   COMMAND
+  ${CMAKE_COMMAND} -E env ZEPHYR_BASE=${ZEPHYR_BASE}
```

Comment:
Creating processes is slow on Windows unfortunately. Let's set this once in boilerplate.cmake instead of on each execute_process that uses the env var.

set(ENV{ZEPHYR_BASE} ${ZEPHYR_BASE})

Reply:
thanks for the comment, and I agree with you, this is not the nicest solution.
However, I also dislike exporting environment variables on global scale and allow all parts of the build system to rely on this being set.

The right fix in this case, imho, is to update `kconfig.py` so that zephyr base can be specified as argument.
Will do that instead, unless you have good reasons not to do so.

Reply:
as a small note, this was the only place I found where ZEPHYR_BASE is required in env in order to work.

Reply:
Specifying zephyr base on the CLI is OK

Reply:
NB: This is still not fixed.

Reply:
I know, it is a little trickier than first anticipated, so haven't had the time for this part yet.

Reply:
@SebastianBoe Fixed.